packages/components/inputs/localized-multiline-text-input/src/localized-multiline-text-input.tsx

```diff
@@ -125,11 +255,15 @@ const LocalizedMultilineTextInput = (props) => {
               <TranslationInput
                 key={language}
                 autoComplete={props.autoComplete}
-                id={LocalizedMultilineTextInput.getId(props.id, language)}
+                id={LocalizedMultilineTextInput.getId(
+                  props.id,
+                  language as string
```

Comment:
I'm wondering why we need to explicitly say `language` is a string when here we're iterating over the `languages` constants which has been actually defined as an array of strings.